package.json

```diff
@@ -55,9 +55,9 @@
         "substs": "https://github.com/yunxing/substs.git",
         "easy-format": "https://github.com/npm-opam/easy-format",
         "merlin-extend": "https://github.com/npm-opam/merlin-extend",
-        "opam-installer-bin": "https://github.com/yunxing/opam-installer-bin.git",        
+        "opam-installer-bin": "https://github.com/yunxing/opam-installer-bin.git",
         "nopam": "https://github.com/yunxing/nopam.git",
-        "utop": "https://github.com/npm-opam/utop",
+        "utop-bin": "https://github.com/reasonml/utop-bin",
```

Comment:
(That would require that `utop-bin`'s `package.json` has its name specified as `"utop"` even if the repo is named `utop-bin`).